controllers/issue.js

```diff
@@ -260,8 +260,8 @@ exports.view = function (aReq, aRes, aNext) {
       }
 
       // Discussion
-      discussion = options.discussion = modelParser.parseDiscussion(aDiscussion);
-      modelParser.parseIssue(discussion);
+      options.discussion = discussion = modelParser.parseDiscussion(aDiscussion);
+      modelParser.parseIssue(discussion); // WARNING: Return value never used
```

Comment:
> So all the assingments are applied onto the object supplied in the parameter.

Interesting feature of *mongoose*. Thanks. I just checked if the line is needed and it is.. otherwise `options.discussion.issueCloseUrl` becomes undefined... so I'll probably add the return value for consistency in case things change down the line.

> will only return a new object if it's still a mongoose Document (with `discussion.toObject()`).

That particular function typically makes a copy of the mongoose object thus removes any linkage to *mongoose* from my experience... so did you mean "with *out*"?